src/docs/js-helpers/classnames.mdx

```diff
@@ -0,0 +1,37 @@
+---
+name: Classnames
+menu: JS Helpers
+route: /js-helpers/class-names
+---
+
+# Classnames
+
+import { Playground } from 'docz'
+import { classNames} from '../../lib';
+
+The `classNames` helper joins all classnames you pass into the function as
+single classname. It automatically filters out empty strings and values that
```

Comment:
> values that are not strings

Hmm, but `styles.root` is an object, isn't it…? 🤔

Reply:
@adamkudrna It not an object. If you import and use something like `styles.root` from SCSS file, webpack loader generates styles and at the same time, it replaces object with string so that `styles.root` becomes a classname. It the magic performed by loaders.